spring-cloud-generator/WORKSPACE

```diff
@@ -0,0 +1,72 @@
+workspace(name = "spring_cloud_generator")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+
+#googleapis
+
+_googleapis_commit = "7438480b2a1bc6371d748e974f7a3647f90c4e8d"
```

Comment:
What is this googleapis commit use for? I thought for the actual generation process we are going to dynamically compute the committish for each library?

Reply:
You're right, I accidentally left this in from copying gapic-generator-java setup but it should be unused in the spring code generation process. Will remove.

Reply:
Removed in [d9bb09f](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1686/commits/d9bb09f1bc7eb9edd5e35ce0ed61028b866ac54e).